mopidy_musicbox_webclient/static/js/gui.js

```diff
@@ -462,6 +451,8 @@ $(document).ready(function(event) {
 
     $(window).hashchange();
 
+    progressTimer = new ProgressTimer({callback: timerCallback});
```

Comment:
This can just be `new ProgressTimer(timerCallback)` if all you are setting is the callback :-)